api/python/notebooks/analysis_demo/comp_bio_query_data_and_metadata.ipynb

```diff
@@ -0,0 +1,1056 @@
+       "2086595          colon                  UBERON:0001155  \n",
+       "2086596          colon                  UBERON:0001155  \n",
+       "\n",
+       "[2086597 rows x 21 columns]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cell_metadata_all_unknown_sex = census[\"census_data\"][\"homo_sapiens\"].obs.read_as_pandas_all(\n",
+    "    value_filter = \"sex == 'unknown'\"\n",
+    ")\n",
+    "\n",
+    "cell_metadata_all_unknown_sex"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can use both `column_names` and `value_filter` to perform specific queries. For example let's fetch the `disease` columns for the `cell_type` `\"B cell\"` in the `tissue_general` `\"lung\"`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "disease                                cell_type  tissue_general\n",
+       "lung adenocarcinoma                    B cell     lung              50228\n",
+       "non-small cell lung carcinoma          B cell     lung              17484\n",
+       "normal                                 B cell     lung              15081\n",
+       "squamous cell lung carcinoma           B cell     lung              11584\n",
+       "chronic obstructive pulmonary disease  B cell     lung               7147\n",
+       "interstitial lung disease 2            B cell     lung               5141\n",
+       "interstitial lung disease              B cell     lung               1655\n",
+       "COVID-19                               B cell     lung                704\n",
+       "small cell lung carcinoma              B cell     lung                583\n",
+       "non-specific interstitial pneumonia    B cell     lung                284\n",
+       "hypersensitivity pneumonitis           B cell     lung                 13\n",
+       "sarcoidosis                            B cell     lung                  6\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cell_metadata_b_cell = census[\"census_data\"][\"homo_sapiens\"].obs.read_as_pandas_all(\n",
+    "    value_filter = \"cell_type == 'B cell' and tissue_general == 'lung'\",\n",
+    "    column_names = [\"disease\"],\n",
+    ")\n",
+    "\n",
+    "cell_metadata_b_cell.value_counts()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Querying gene metadata (var)\n",
+    "\n",
+    "The human gene metadata of the Cell Census is located at `census[\"census_data\"][\"homo_sapiens\"].ms[\"RNA\"].var`. Similarly to the cell metadata, it is a `SOMADataFrame` and thus we can also use its method `read_as_pandas_all()`. \n",
```

Comment:
Nit:  do we need to clarify the contents of this measurement?  I.e., "human gene metadata of the Cell Census, for RNA assays, is located ..."

Reply:
Added